tests/integration.rs

```diff
@@ -2032,8 +2032,14 @@ mod parallel {
                 &event_path
             ));
 
+            // It's been observed on the Bionic image that udev and snapd
+            // services can cause some delay in the VM's shutdown. Disabling
+            // them improves the reliability of this test.
+            guest.ssh_command("sudo systemctl disable udev").unwrap();
+            guest.ssh_command("sudo systemctl disable snapd").unwrap();
```

Comment:
I don't think `disable` will actually stop the services if they're already running. From the man page:

           This command implicitly reloads the system manager configuration
           after completing the operation. Note that this command does not
           implicitly stop the units that are being disabled. If this is
           desired, either combine this command with the --now switch, or
           invoke the stop command with appropriate arguments later.

Reply:
Ok so should I also invoke `sudo systemctl stop udev` and `sudo systemctl stop snapd` after I've disabled them?